### PR TITLE
Spelling, grammar, and related consistency improvements

### PR DESCRIPTION
**Pull request by DomenicF**
# Changed
- Changed: Updated many spelling errors and wordings to be more clear

### CODE REVIEW

**Comment by DomenicF** [comment]
Hi - just checking in. Any issues or comments?

File: UI/Web/src/app/admin/manage-email-settings/manage-email-settings.component.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-            Registration/Confirmation emails will not be sent if you are not accessing Kavita via a publicly reachable url.
-            <span class="text-warning">If you want Send To device to work, you must host your own email service.</span>
+        <p>Kavita comes out of the box with an email service to power tasks like inviting users, password reset requests, etc. Emails sent via our service are deleted immediately. You can use your own
+            email service by setting up the <a href="https://github.com/Kareadita/KavitaEmail" target="_blank" rel="noopener noreferrer">Kavita Email</a> service. Set the URL of the email service and use the Test button to ensure it works.
+            You can reset these settings to default at any time. There is no way to disable emails for authentication, although you are not required to use a
+            valid email address for users. Confirmation links will always be saved to logs and presented in the UI.
+            Registration/confirmation emails will not be sent if you are not accessing Kavita via a publicly reachable URL.
```
Comment (majora2007) [inline UI/Web/src/app/admin/manage-email-settings/manage-email-settings.component.html:8]: Can you reword this in retrospect to include that:
Registration/confirmation emails will not be sent if you are not accessing Kavita via a publicly reachable URL or unless Host Name is configured.

Host name is a new feature that allows you to specify the reverse proxy used to access Kavita and hence Kavita will assume that for email generation and override the check if accessible.

**Review by majora2007 — CHANGES_REQUESTED** [review]
Just one small enhancement to reflect some new features in Kavita and I can merge.

**Comment by DomenicF** [comment]
Done. Let me know if that's what you wanted or if I misunderstood.

**Review by majora2007 — APPROVED** [review]

**Pull request merged**